jenkins/Jenkinsfile-build

```diff
@@ -0,0 +1,11 @@
+library('private-pipeline-library')
+library('jenkins-shared')
+
+mavenSnapshotPipeline(performSonarAnalysis: true,
```

Comment:
Just to confirm:  The bamboo build is configure for java 6 and maven 3.0.x    And this config will do java 8 and a much more recent version of maven3? Is that okay?

Reply:
I think that as long as the build continues to work with the org-wide default build tools it is okay. So far, only one build has required that I revert to older jdk / maven.

Reply:
Work for me.  From the bamboo job, I wasn't sure if the older versions were required, e.g. there was some weird jvm based bug that necessitated older versions.  

Ship it!

        /|\
      /_|_\
 ____|____
 \_o_o_o_/
~~ |     ~~~~~
___t_________